### PR TITLE
OCD-3443 - tests for /hierarchy end point

### PR DESCRIPTION
**Pull request by athomas-git**

### CODE REVIEW

File: chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json [hunk 1]
```diff
@@ -1,98 +1,508 @@
 {
 	"info": {
-		"_postman_id": "21c26d32-3ea4-4d99-ac3d-c4c557baa398",
+		"_postman_id": "ec8c7252-3caa-48d3-8060-d247a94d6ea3",
```
Comment (andlar) [inline chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json]: Is this line needed? It seems to change often, but I don't know what it's used for. If it's metadata kinds of stuff do we need to track it at all?

File: chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json [hunk 2]
```diff
@@ -1,98 +1,508 @@
...
+									"pm.test(\"Valid api key header should return 200 status and response is not  null\", function () {\r",
+									"    pm.response.to.have.status(200);\r",
+									"    var jsonData = pm.response.json();\r",
+									"    pm.expect(jsonData.response).not.eq(null);\r",
+									"});\r",
+									"pm.test(\"Developer Names are correct as per the id passed in\", function () {\r",
+									"    var actualBody= pm.response.json();\r",
+									"    var expected= pm.iterationData.get(\"name\");\r",
+									"    pm.expect(actualBody.name).to.equal(expected);\r",
+									"    var expected= pm.iterationData.get(\"developerCode\");\r",
```
Comment (andlar) [inline chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json]: Does this line redefine the `expected` variable? It shouldn't do that

File: chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json [hunk 3]
```diff
@@ -1,98 +1,508 @@
...
+									"      \"acbName\": \"string\",\r",
+									"      \"attestation\": {\r",
+									"        \"removed\": false,\r",
+									"        \"transparencyAttestation\": \"string\"\r",
+									"      }\r",
+									"    }\r",
+									"  ],\r",
+									"  \"website\": \"string\"\r",
+									"};\r",
+									"pm.test(\"Validate that /heirarchy end point has correct schema\", () => {\r",
```
Comment (andlar) [inline chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json]:
```suggestion
									"pm.test(\"Validate that /hierarchy end point has correct schema\", () => {\r",
```

File: chpl/chpl-api/e2e/script/newman-run-dev.js [hunk 4]
```diff
@@ -4,6 +4,7 @@ const path = require('path');
 
 const collection_path = path.resolve('chpl/chpl-api/e2e/collections');
 const reports_path = path.resolve('newman');
+const data_path= path.resolve('chpl/chpl-api/e2e/data');
```
Comment (andlar) [inline chpl/chpl-api/e2e/script/newman-run-dev.js]:
```suggestion
const data_path = path.resolve('chpl/chpl-api/e2e/data');
```

File: chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json (hunk 1 above)
Comment (athomas-git) [reply chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json]: Tests are running successfully without postman_id. So removed it.

File: chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json (hunk 2 above)
Comment (athomas-git) [reply chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json]: This is done

File: chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json (hunk 3 above)
Comment (athomas-git) [reply chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json]: This is done

File: chpl/chpl-api/e2e/script/newman-run-dev.js (hunk 4 above)
Comment (athomas-git) [reply chpl/chpl-api/e2e/script/newman-run-dev.js]: This is done

**Review by andlar — DISMISSED** [review]

**Review by tmy1313 — DISMISSED** [review]

**Review by palakpatel311 — DISMISSED** [review]

**Review by ashwinimore — DISMISSED** [review]

**Review by palakpatel311 — APPROVED** [review]

**Review by ashwinimore — APPROVED** [review]

**Pull request merged**